scripts/_check_docs_utils.py

```diff
@@ -2,7 +2,7 @@
 
 # The following code was adapted from PyCQA for use
 # in the Oppia project. See
-# https://github.com/PyCQA/pylint/blob/master/pylint/extensions/_check_docs_utils.py 
+# https://github.com/PyCQA/pylint/blob/master/pylint/extensions/_check_docs_utils.py
```

Comment:
Hi @seanlip! Can you please have a look at this? Since we have adopted this code from PyCQA, I have added this after discussing with @kevinlee12. Pylint has a GNU GPL v2.0 license.